api/overview/azure/devices.md

```diff
@@ -1,83 +0,0 @@
----

```

Comment:
I would check with @Campsoper on this - not sure we can just delete a file, but you may have already looked into this